seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/client/StarRocksStreamLoadVisitor.java

```diff
@@ -165,12 +167,12 @@ private void checkLabelState(String host, String label) throws IOException {
                 String queryLoadStateUrl = new StringBuilder(host).append("/api/").append(sinkConfig.getDatabase()).append("/get_load_state?label=").append(label).toString();
                 Map<String, Object> result = httpHelper.doHttpGet(queryLoadStateUrl, getLoadStateHttpHeader(label));
                 if (result == null) {
-                    throw new IOException(String.format("Failed to flush data to StarRocks, Error " +
+                    throw new StarRocksConnectorException(CommonErrorCode.ILLEGAL_ARGUMENT, String.format("Failed to flush data to StarRocks, Error " +
```

Comment:
Why use CommonErrorCode.ILLEGAL_ARGUMENT？

Reply:
I am very confused about the same ILLEGAL_ARGUMENT having diff places, with the same meaning;

Reply:
You can use the prompts to know which error code to use.In this case, the error code should be related to `flush data fail`.